geom/src/circle.rs

```diff
@@ -42,9 +42,8 @@ impl Circle {
         self.to_ring().into_polygon()
     }
 
-    /// Renders some percent, between [0, 1], of the circle as a polygon. The polygon starts from 0
-    /// degrees. Be warned the resulting polygon doesn't have a ring as its boundary!
-    pub fn to_partial_polygon(&self, percent_full: f64) -> Polygon {
+    /// Renders some percent, between [0, 1], of the circle. The shape starts from 0 degrees.
+    pub fn to_partial_tessellation(&self, percent_full: f64) -> Tessellation {
```

Comment:
It'd take a bit of effort to make a `Ring` out of this. The center point needs to exist, unless `percent_full = 1.0`. And it's valid to pass `0` in, in which case the tessellation is empty. That's no good for a Ring, so it'd require an API change anyway. All of the callers just draw this.